openmetadata-ui/src/main/resources/ui/src/components/Webhooks/webhookV1.less

```diff
@@ -0,0 +1,24 @@
+/*
+ *  Copyright 2022 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+@import '../../styles/antd-master.less';
```

Comment:
We don't need this
```suggestion

```

Reply:
i tried without this, its throws an error, as I am accessing primary color